install-dev/upgrade/sql/1.7.8.0.sql

```diff
@@ -21,7 +21,9 @@ INSERT IGNORE INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `po
   (NULL, 'actionFrontControllerSetVariables', 'Add variables in JavaScript object and Smarty templates', 'Add variables to javascript object that is available in Front Office. These are also available in smarty templates in modules.your_module_name.', '1'),
   (NULL, 'displayAdminGridTableBefore', 'Display before Grid table', 'This hook adds new blocks before Grid component table.', '1'),
   (NULL, 'displayAdminGridTableAfter', 'Display after Grid table', 'This hook adds new blocks after Grid component table.', '1'),
-  (NULL, 'displayAdminOrderCreateExtraButtons', 'Add buttons on the create order page dropdown', 'Add buttons on the create order page dropdown', '1')
+  (NULL, 'displayAdminOrderCreateExtraButtons', 'Add buttons on the create order page dropdown', 'Add buttons on the create order page dropdown', '1'),
+  (NULL, 'actionAdminSecurityControllerPostProcessBefore', 'On post-process in Admin Security Controller', 'This hook is called on Admin Security Controller post-process before processing any form', '1'),
+  (NULL, 'actionAdminSecurityControllerPostProcessGeneralBefore', 'On post-process in Admin Security Controller', 'This hook is called on Admin Improve Shipping Preferences post-process before processing the General form', '1')
 ;
```

Comment:
You need to add new Tab in upgrade script as well you can inspire from this
https://github.com/PrestaShop/PrestaShop/blob/develop/install-dev/upgrade/php/ps_1760_update_tabs.php

But be careful there was a fix here https://github.com/PrestaShop/PrestaShop/blob/develop/install-dev/upgrade/php/ps_1763_update_tabs.php

But by merging both you should find the proper way 😅